cawg_identity/src/tests/builder/simple_case.rs

```diff
@@ -66,17 +67,17 @@ async fn simple_case() {
     assert_eq!(manifest_store.validation_status(), None);
 
     let manifest = manifest_store.active_manifest().unwrap();
-    let mut ia_iter = IdentityAssertion::from_manifest(manifest);
+    let mut st = StatusTracker::default();
+    let mut ia_iter = IdentityAssertion::from_manifest(manifest, &mut st);
```

Comment:
Ack - StatusTracker becomes a parameter we carry around in most calls. Correct?

Reply:
Yes, this is in line with the general approach to handling errors in C2PA parsing and validation.